src/pages/workspaces/workspace/analysis/modals/CromwellModal.js

```diff
@@ -10,7 +10,8 @@ import Events, { extractWorkspaceDetails } from 'src/libs/events'
 import { withDisplayName } from 'src/libs/react-utils'
 import * as Utils from 'src/libs/utils'
 import { computeStyles, getCurrentApp, getCurrentAppDataDisk } from 'src/pages/workspaces/workspace/analysis/runtime-utils'
-import { tools } from 'src/pages/workspaces/workspace/analysis/tool-utils'
+
+import { appTools } from '../tool-utils'
```

Comment:
Why is it necessary to do a relative import? I don't think we do this anywhere else in the codebase.

Reply:
I'd like to bring this up for discussion.  Relative paths for very nearby related files makes the code more portable.  For things that are not nearby, yes absolute pathing is great.

But this will also become an issue as we look at breaking some things up into shared packages... absolute paths take some extra tooling work to not cause problems for the package consumer of a shared package.  This is an issue I solved for in a previous project.

Reply:
This is an accident, IDE likely imported it that way. I'm planning to revert it since it seems a bit out of scope to make this decision on this PR.